packages/oui-calendar/_normalize.less

```diff
@@ -44,6 +44,31 @@
     &.rightMost::after { transform: translateX(50%); }
 
     &.inline { box-shadow: none; }
+
+    // Inputs
+    .numInputWrapper {
+      &:hover { background: none; }
+
+      input {
+        font-size: @oui-calendar-font-size;
+        font-weight: @oui-calendar-font-weight;
+        color: @oui-calendar-input-font-color;
+        background: @oui-calendar-input-background-color;
+
+        &:hover,
+        &:focus { background-color: @oui-calendar-input-background-color_hover; }
+      }
+
+      span {
+        background-color: @oui-calendar-input-background-color;
```

Comment:
```suggestion
   span {
         &,
         &:hover,
￼         &:focus { background-color: @oui-calendar-input-background-color; }
   }
```

Reply:
Yes, could be. There was two different colors before, but it has been changed.